Small tweaks to intro navigation for a more intuitive flow (#11784)
getsentry/sentry-docs: docs/product/sentry-basics/index.mdx

**Want a TL;DR for getting started? Download our ["Developer Quick Reference Guide"](/pdfs/developer-quick-reference-guide.pdf)**

> ## Tutorials: Distributed Tracing, Frontend and Backend Error Monitoring

Reviewer: ```suggestion
## Tutorials: How to Set Up Tracing and Error Monitoring
```
